jetson/percep/pcl.hpp

```diff
@@ -52,7 +52,7 @@ class PCL {
     private:
 
         //Filters points with values beyond certain threshold
-        void PassThroughFilter();
+        void PassThroughFilter(const std::string axis, const double upperLimit);
```

Comment:
`passThroughFilter` (lower case "p")? Might be worth a general cleanup of the entire CV project (in a separate issue, not this one).

Reply:
We'll address that in another issue

Reply:
Link to the issue?

Reply:
https://github.com/umrover/mrover-workspace/issues/516